sdk/communication/AzureCommunicationChat/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.0.2 (TODO add date)
+### Key Bug Fixes
```

Comment:
This needs to be Bugs Fixed to align with updated Changelog guidance:
https://azure.github.io/azure-sdk/policies_releases.html#change-logs